src/test/java/edu/ksu/canvas/attendance/services/AttendanceSummaryCSVServiceUTest.java

```diff
@@ -27,8 +27,11 @@
     private final int ARBITRARY_STUDENT_MISSED_1 = 2;
     private final int ARBITRARY_STUDENT_TOT_MADEUP_1 = 2;
     private final int ARBITRARY_STUDENT_TOBE_MADEUP_1 = 3;
+    private final int ARBITRARY_STUDENT_EXCUSED_1 = 3;
+    private final int ARBITRARY_STUDENT_PRESENT_1 = 1;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1170)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "ARBITRARY_STUDENT_PRESENT_1" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00116)